### ISSUE TITLE
Audit the HTML and accessibility quiz

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/freeCodeCamp#58308]
# Notes from Audit

-  `What is accessibility?` options should be updated. All of the wrong answers are shorter making it more obvious what the correct answer is.
- Update question and options for `What is a best practice for creating accessible headings?`
- the `What is WAI-ARIA?` question is pretty similar to the `What is accessibility?` question. So that can be updated to instead ask what WAI-ARIA stands for  
- the `What is the function of the hidden attribute?` is covered more in later modules. So this question should be updated to instead ask about the `aria-hidden` attribute 
- the distractors are to short which makes it obvious what the correct answer is for the `What is the function of the `tabindex` attribute?`
- the options for accesible tables question can be updated
- the `aria-selected` question can be rephrased to provide more variety in the quiz 
- the `aria-expanded` question can also be updated to provide more variety in the quiz so the question formats aren't as predictable
- the options for the `What is the correct way to give an input a label?` question need to be updated because the correct answer is much longer than the wrong answers
- this question doesn't seem to be explicitly covered should it should be updated: `What is a common keystroke to see if a website is keyboard navigation-friendly?`
- the `aria-live` attribute question options need to be updated because the correct answer is twice as long as the wrong ones
- the correct answer for the `What is the purpose of the `role` attribute?` is twice as long as the wrong answer making is super obvious which one to pick
- the options for the `What is the best way to make a website's link accessible?` need to be updated
- the correct answer for `When recording audio for a video, it is best to:` is twice as long as the wrong ones. 
- the options for the `What is the role of the `aria-checked` attribute?` need to be updated because the correct answer is twice as long as the wrong ones making it obvious which one to choose.
